src/main/server.ts

```diff
@@ -15,10 +17,10 @@ if (app.locals.ENV === 'development' || app.locals.ENV === 'dockertests') {
   }
   const server = https.createServer(serverOptions, app)
   server.listen(port, () => {
-    console.log(`Application started: https://localhost:${port}/legal`)
+    logger.info(`Application started: https://localhost:${port}`)
```

Comment:
you may want to grab rafal's latest changes in this file. also I don't think you meant to remove `/legal`

Reply:
we need legal

Reply:
I've put back legal but will leave the updating the file to a separate PR as need this merge asap